tox.ini

```diff
@@ -6,11 +6,10 @@ deps =
     coverage
     pretend
     pytest
-    https://github.com/pypa/pip/archive/master.zip#egg=pip
-# The --ignore-installed install options is needed to install pip url
-# (needed to issue #95). This can be removed once pip 10.0 is out.
+    pip
+
```

Comment:
The switch to master was to get dd3defe509d6103605d8da8270a4787f0e8ec616, released in 9.0.2, so `pip>=9.0.2` might be better. Also, you added an extraneous empty line.

Reply:
thanks! fixing